cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -139,10 +143,11 @@ func generateMountsModeInstall(mst *initramfsMountsState) error {
 	if err := modeEnv.WriteTo(boot.InitramfsWritableDir); err != nil {
 		return err
 	}
-	// we need to put the file to disable cloud-init in the
-	// _writable_defaults dir for writable-paths(5) to install it properly
-	writableDefaultsDir := sysconfig.WritableDefaultsDir(boot.InitramfsWritableDir)
-	if err := sysconfig.DisableCloudInit(writableDefaultsDir); err != nil {
+
+	opts := &sysconfig.Options{
+		TargetRootDir: boot.InitramfsWritableDir,
```

Comment:
we should apply the gadget also install mode or turn off console conf there in some other way, or both?

Reply:
Well console-conf is itself already disabled on install mode anyways but I suppose I could see an argument for the other settings to still take effect in install mode anyways like disabling ssh or setting the power-button-action, etc.

Reply:
done

Reply:
how is it disabled in install mode (I don't remember, is not obvious from the service file)

Reply:
it is disabled in install mode via https://github.com/CanonicalLtd/subiquity/blob/fe012f20dc758f1fb1641d0b5744baf00739524d/bin/console-conf-wrapper#L25